src/main/scala/com/gu/support/workers/model/states/CreatePaymentMethodState.scala

```diff
@@ -11,8 +11,4 @@ case class CreatePaymentMethodState(
   paymentFields: PaymentFields,
   acquisitionData: Option[AcquisitionData],
   sessionId: Option[String]
-) extends StepFunctionUserState {
-
-  def accessScope: AccessScope = AccessScope.fromRaw(sessionId)
```

Comment:
this function was only actually used on the zuora state, so I just removed from here